cmd/deploy/start.go

```diff
@@ -238,6 +248,7 @@ func WithLocalFile(cmd *cobra.Command, options *deployStartOptions, deployClient
 		UnstructuredDeployment:  payload,
 		ApplicationNameOverride: options.application,
 		Context:                 options.context,
+		Scmc:                    scmc,
```

Comment:
```suggestion
		SCMC:                    scmc,
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
I'm not gonna copy paste this comment everywhere but `Scmc` should be `SCMC` or `scmc` everywhere.

Reply:
Changed all Scmc references to SCMC